plugins/jobs/src/js/components/JobDeleteModal.js

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import { Confirm } from "reactjs-components";
+import StringUtil from "#SRC/js/utils/StringUtil";
+import UserActions from "#SRC/js/constants/UserActions";
+
+function modalLabel(jobId, stopCurrentJobRuns) {
+  const defaultLabel = `${StringUtil.capitalize(UserActions.DELETE)} Job`;
+
+  const stopCurrentRunsLabel = `Stop Current Runs and ${StringUtil.capitalize(
+    UserActions.DELETE
+  )} Job`;
+
+  return stopCurrentJobRuns ? stopCurrentRunsLabel : defaultLabel;
+}
+
+const ModalMessage = ({ jobId, stopCurrentJobRuns }) => {
+  const defaultMessage = `Are you sure you want to ${
+    UserActions.DELETE
+  } ${jobId}? This action is irreversible.`;
+
+  const stopCurrentRunsMessage = `Couldn't ${
+    UserActions.DELETE
```

Comment:
was this prettier? 😱

Reply:
Yes.

Reply:
Prettier would be fine with a multiline string. Dunno if our component would be fine with it though 🤔 
```js
const stopCurrentRunsMessage = `Couldn't ${UserActions.DELETE} ${jobId} as there
 are currently active job runs. Do you want to stop all runs and
 ${UserActions.DELETE} the job?`;
```

Reply:
When I run prettier, it reverts it back. Might be something with my editor.